src/test/java/suite/core/CoreTest.java

```diff
@@ -0,0 +1,69 @@
+package suite.core;
+
+import app.common.HttpClient;
+import app.common.Mapper;
+import app.component.Core;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.uid2.shared.attest.JwtService;
+import com.uid2.shared.attest.JwtValidationResponse;
+import io.vertx.core.json.JsonObject;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CoreTest {
+    @ParameterizedTest(name = "/attest - {0}")
+    @MethodSource({
+            "suite.core.TestData#attestArgs"
+    })
+    public void testAttest_EmptyAttestationRequest(Core core) {
+        HttpClient.HttpException exception = assertThrows(
+                HttpClient.HttpException.class,
+                () -> core.attest("")
+        );
+
+        String coreUrl = core.getBaseUrl();
+
+        assertEquals("Unsuccessful POST request - URL: " + coreUrl + "/attest - Code: 400 Bad Request - Response body: {\"status\":\"no attestation_request attached\"}", exception.getMessage());
+    }
+
+    @ParameterizedTest(name = "/attest - {0}")
+    @MethodSource({
+            "suite.core.TestData#attestArgs"
+    })
+    public void testAttest_ValidAttestationRequest(Core core) throws Exception {
+        final String validTrustedAttestationRequest = "{\"attestation_request\":\"AA==\",\"public_key\":\"MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAl7kXK1wf15V9HgkQhbMK2nfJGbudmdXNrX7MZjFm07z6eEjaQsuqMteQumLRwn+RxEcXKVaxBAE3RQTFL9XsZc2OtRKOU+oMIQep8tmPFMgh83BzjLs5O5HZf510geFJO6qRqc3UHJT3ACxE7IkmRx1JIKFKPzTrthHdb2+D7bdJBPsVbwk7y+a36f5jvELGGzMC89LAvd7JpOmGsCAj6jwiEAKmmLId9bfe0YeLuebl95VfSzrQVdz82oGGQKXuJgKZtc/Xp1omZ9spm+zzVFJzsimxDdGdnaWMnas43VoTE04JDt+pucJTTbftIvu05frwkbZh3sQ2yBu5gBP7YwIDAQAB\",\"application_name\":\"uid2-operator\",\"application_version\":\"5.27.10-3f25586306\",\"components\":{\"uid2-attestation-api\":\"2.0.0-f968aec0e3\",\"uid2-shared\":\"7.2.4-SNAPSHOT\"}}";
+
+        String attestationResponse = core.attest(validTrustedAttestationRequest);
+
+        assertNotNull(attestationResponse);
+        JsonNode response = Mapper.OBJECT_MAPPER.readTree(attestationResponse);
+        assertNotNull(response.get("status"));
```

Comment:
What do you think about directly doing `assertEquals("success", response.get("status").asText())`? Since if `status` is null, this assertion should fail anyway

Reply:
I think it is better to be explicit in what you are testing for. A response that is not 'success' is different to a null response